scripts/pre_commit_linter.py

```diff
@@ -202,6 +202,16 @@
             'core/tests/performance_framework/perf_domain.py'),
         'excluded_dirs': ('scripts/',)
     },
+    {
+        'regexp': r'# pylint: disable=[A-Z][0-9]{4}',
+        'message': "Please remove this pylint pragma if it is unnecessary or "
```

Comment:
You need to add a `+` to concatenate the multi-line message string. Did you manually test this check by deliberately using a statement which matches with the regexp?

Reply:
Yes I did check the test by manually using a statement and it works. And regarding the concatenating the strings can you please tell me why  ```+``` should be used as when I tested it printed all lines successfully.

Reply:
I used the online editor as there were merge conflicts and aren't the tests run on the Travis after pushing a commit?

Reply:
Regarding the +, I looked into this and found out that it is used at some places to concatenate the message. Let's not use it here and adopt this as a convention. Therefore no change required there.
Re the online editor, all tests are run on Travis but we prefer to run the pre-submit checks and fix those errors before hand so that no Travis re-run is required for atleast those errors.
Thanks!

Reply:
You don't need a '+', but please use single quotes throughout. (@apb7 didn't we have a lint check for this?)

Reply:
@seanlip done. Should I open an issue regarding the single quotes?

Reply:
Yes please. Thanks!

Reply:
@lilithxxx has filed issue #5663. I have added some more context there. Thanks!